src/sentry/monitors/endpoints/organization_monitor_stats.py

```diff
@@ -33,8 +34,16 @@ def get(self, request: Request, organization, project, monitor) -> Response:
             status__in=[CheckInStatus.OK, CheckInStatus.ERROR, CheckInStatus.MISSED],
             date_added__gt=args["start"],
             date_added__lte=args["end"],
-        ).values_list("date_added", "status", "duration")
-        for datetime, status, duration in history.iterator():
+        )
+
+        environments = get_environments(request, organization)
+
+        if environments:
+            history = history.filter(monitor_environment__environment__in=environments)
```

Comment:
When filter on multiple environments what will this all look like?

Reply:
see the test case. right now it presents them aggregated. floated this earlier. think we should keep it simple for now to ship

Reply:
👍